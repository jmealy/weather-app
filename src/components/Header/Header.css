.Header {
  background-image: url('../../assets/pattern.svg');
}