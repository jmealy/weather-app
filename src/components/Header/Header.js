import React from 'react';
import LocationSearch from '../LocationSearch/LocationSearch'
import './Header.css';

const Header = () => {
  return (
    <div className="Header">
      <LocationSearch></LocationSearch>
    </div>
  );
}

export default Header;
