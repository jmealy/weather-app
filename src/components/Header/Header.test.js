import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('renders learn react link', () => {
  const onLocationSubmit = jest.fn();
  const tree = renderer.create(<Header onLocationSubmit={onLocationSubmit} />);
  expect(tree).toMatchSnapshot();
});
