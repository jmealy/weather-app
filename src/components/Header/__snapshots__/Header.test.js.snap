// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="header"
>
  <div
    className="search"
  >
    <h1>
      Enter a City
    </h1>
    <div
      className="searchForm"
    >
      <form
        className="formContainer"
        onSubmit={[Function]}
      >
        <input
          className="textInput"
          placeholder="London"
          type="text"
        />
        <input
          className="submitButton"
          type="submit"
          value="Get Weather"
        />
      </form>
    </div>
  </div>
</div>
`;
